rdr_service/api/mail_kit_order_api.py

```diff
@@ -30,7 +30,7 @@ def _lookup_resource_type_method(resource_type_method_map, raw_resource):
         except KeyError:
             raise MethodNotAllowed("Method not allowed for resource type {}".format(resource_type))
 
-    @auth_required(PTC)
+    @auth_required(PTC_AND_PPSC)
     def post(self):
```

Comment:
I may be misremembering, but I think these endpoints are set up to use POST for creating mail-kit orders and PUT for updating them. Will PPSC not need access to the PUT endpoint?

Reply:
Good catch, I missed the PUT endpoints. will update in a moment.